Examples/Swift/ShowHideLayerExample.swift

```diff
@@ -26,14 +26,14 @@ class ShowHideLayerExample_Swift: UIViewController, MGLMapViewDelegate {
     }
     
     func addLayer(to style: MGLStyle) {
-        let source = MGLVectorSource(identifier: "contours", configurationURL: NSURL(string: "mapbox://mapbox.mapbox-terrain-v2")! as URL)
+        let source = MGLVectorTileSource(identifier: "contours", configurationURL: NSURL(string: "mapbox://mapbox.mapbox-terrain-v2")! as URL)
         
         let layer = MGLLineStyleLayer(identifier: "contours", source: source)
         layer.sourceLayerIdentifier = "contour"
-        layer.lineJoin = MGLStyleValue(rawValue: NSValue(mglLineJoin: .round))
-        layer.lineCap = MGLStyleValue(rawValue: NSValue(mglLineCap: .round))
-        layer.lineColor = MGLStyleValue(rawValue: UIColor.brown)
-        layer.lineWidth = MGLStyleValue(rawValue: 1.0)
+        layer.lineJoin = NSExpression(forConstantValue: NSValue(mglLineJoin: .round))
+        layer.lineCap = NSExpression(forConstantValue: NSValue(mglLineJoin: .round))
```

Comment:
`lineCap` is being set to `MGLLineJoin.round` instead of `MGLLineCap.round`. Note that there’s no guarantee that `MGLLineJoin.round` has the same underlying value as `MGLLineCap.round`, although that does happen to be the case. The compiler is unable to ensure that the right type of constant value is being set on the property, so one advantage of using a string literal here is that `"round"` is `"round"` no matter which property it’s being set on.